2024/Autumn/GSoC-Intro-Seminar/README.md

```diff
@@ -0,0 +1,9 @@
+# Google Summer of Code 2025 Intro Seminar
+
+**20th October 2024, 2:00pm, NR121**
+
+**Speakers:** 
+1. **Rohan Barsagade** @Wellcome Sanger Tree of Life, *GSoC'24*
+1. **Ankit Meda** @NumFOCUS, *GSoC'24*
```

Comment:
```suggestion
2. **Ankit Meda** @NumFOCUS, *GSoC'24*
```

Reply:
LIF slides also need to be updated

Reply:
![image](https://github.com/user-attachments/assets/a3468892-4fb5-4900-8d84-7b8433a99fae)

markdown formats that correctly on its own

Reply:
>  LIF slides also need to be updated

this is a separate issue not meant for this channel or this pr